lib/components/component.dart

```diff
@@ -128,6 +130,11 @@ abstract class PositionComponent extends Component {
     final double dy = -anchor.relativePosition.dy * height;
     canvas.translate(dx, dy);
 
+    // Handle inverted rendering by moving center and flipping.
+    canvas.translate(width / 2, height / 2);
```

Comment:
If both `renderFlipX` and `renderFlipY` are false we should avoid executing all of this code.

Reply:
Sounds good, 👍